content/geometry/General.adoc

```diff
@@ -4,19 +4,64 @@
 The unit used for object dimensions is meters.
 
 == Coordinate systems
-The local coordinate system is a right-handed coordinate system according to
-ISO 8855 with the axes pointing to the directions mentioned below. For a non-rotated
-coordinate system, the following applies:
+
+ASAM OpenMATERIAL uses the following types of coordinate systems:
+
+* World coordinate system, see <<sec-World-coordinate-system>>
+* Local coordinate system, see <<sec-Local-coordinate-system>>
+
+Both coordinate systems consist of three orthogonal directions associated with
+x-, y-, and z-axes, and a coordinate origin where axes meet. The coordinate
+systems are right-handed Cartesian coordinate systems according to ISO 8855. For
+a non-rotated coordinate system, the following applies:
 
 * Forward matches x-axis
 * Left matches y-axis
 * Up matches z-axis
 
-NOTE: TODO: Add images similar to OpenDRIVE? See
-https://publications.pages.asam.net/standards/ASAM_OpenDRIVE/ASAM_OpenDRIVE_Specification/latest/specification/08_coordinate_systems/08_04_local_coordinate_system.html
+The positive rotation is assumed to be counter-clockwise ("right-hand rule").
+Orientation of objects is expressed extrinsically by the yaw, pitch, and roll
+angles derived from the Euler rotation sequence in the order z-axis, then
+y-axis, then x-axis.
+
+[#fig-coordinate-system]
+.Yaw, pitch, and roll angle in an ISO 8855:2011 compliant coordinate system
+image::fig_coordinate_system.svg[img, 1100]
+
```

Comment:
@FKlopfer: Please check whether the image references are correct for the current pipeline configuration